frontend/src/app/testQueue/AoEForm/AoEModalForm.tsx

```diff
@@ -70,7 +67,6 @@ const AoEModalForm = (props: AoEModalProps) => {
         saveCallback={saveCallback}
         isModal={true}
         noValidation={true}
-        formRef={formRef}
```

Comment:
**question (non-blocking):** Was this ref just entirely unused?

Reply:
it was assigned but never used